src/index.js

```diff
@@ -1,2 +1 @@
-import './critical.scss';
-import './non-critical.scss';
```

Comment:
specify SCSS files directly in HTML

Reply:
Hi, vielen Dank! Ich weiß, vor ein paar Wochen bin ich durch Recherchen im Internet auf dieses Plugin gestoßen.

Reply:
Hallo @TonyNguyen137,

wenn du Fragen hast, ich kann dir immer helfen.

Reply:
Hi @webdiscus,
ich bekomme mit dem HtmlBundlerPlugin in der Konsole jede Menge Fehlermeldungen, wenn mein SCSS einen Fehler enthält. Um den Fehler zu finden, muss ich aktuell Zeile für Zeile auskommentieren, was ziemlich mühsam ist. Gibt es dafür eine bessere Lösung? Ich habe im Internet nichts dazu gefunden und dachte, ich frage mal direkt bei dir nach.
<img width="1586" alt="Screenshot 2024-12-09 at 20 59 56" src="https://github.com/user-attachments/assets/b9df2a45-4922-4570-b572-ff6fb481fe37">

Reply:
und hier zum Vegleich, diese Meldung bekomme ich von vite.js plugin 
<img width="516" alt="Screenshot 2024-12-09 at 21 20 54" src="https://github.com/user-attachments/assets/45aaa493-253f-4626-a8db-5031366811c0">

Reply:
@TonyNguyen137 

ich kann nicht den Fehler reproduzieren:
![image](https://github.com/user-attachments/assets/46b4570e-b0aa-4baa-845a-58dbef71266e)

Die Fehlermeldung ist exactly genau.

Kannst du bitte ein Repo mit einer reproduzierbaren Fehlermeldung erstellen?

Reply:
hier https://github.com/TonyNguyen137/webpack-test-2, habe in style.scss einfach ein ' bei utilities weggelassen.
<img width="944" alt="Screenshot 2024-12-09 at 22 08 36" src="https://github.com/user-attachments/assets/8538009e-1b96-4256-9996-f57cd76cd8ae">

Reply:
@TonyNguyen137 

Ich habe gefunden warum so kryptische Fehlermeldung kommt.
Der "schuld" war die `minify: true` option.
Setzte bitte die `minify` option bei solche Fehler auf false.

Wenn ein Fehler kommt dann Webpack generate innere JS Fehlermeldung und drauf kommt noch eine minify Module und "versteht" es nicht und generate noch seine Fehlermeldung...


Ich gucke ob ich was im Plugin abfangen kann und frühe steige bei einer Fehler aus.

Reply:
![image](https://github.com/user-attachments/assets/df13da66-871e-4471-8397-58b3259a5107)

Reply:
@TonyNguyen137 

was soll die `exclude` Option machen? Der Plugin hat diese Option nicht.

```js
exclude: ['src/partials/header.hbs'],
```

Reply:
ich hatte vor paar wochen mit .pug Template herum experimentiert und den header Teil gesplittet, nach dem Build wurde der header Teil immer in header.html umgewandelt und in dist Ordner ausgegeben, was ich nicht wollte. Chat GPT hat das dann vorgeschlagen und dann erschien der Header nicht mehr in dist Ordner. Irgendwie hat es funktioniert :D

Reply:
Bitte nicht dem GPT vertrauen. Er hat viele Halluzinationen, ich weiß, er ist sehr ungenau :-)

Reply:
@TonyNguyen137 

das Problem mit der Fehlermeldung ist in `4.10.2` Version behoben.

Reply:
achhhhh wassssss, du bist der Entwickler von dem Toool. Ohhhhhhhha wie krasssss 🤩